aireview/src/aireview/domain/services/code_analyzer.py

```diff
@@ -32,4 +33,8 @@ def _build_analysis_prompt(self, file: PullRequestFile):
 
 
     def _parse_analysis(self, analysis):
-        return ReviewComment.from_json(analysis)
\ No newline at end of file
+        review_comments: [ReviewComment] = []
+        analysis_comments = extract_json(analysis)
```

Comment:
**documentation**: Consider adding type hints for analysis_comments variable to improve code readability and maintainability.